File: FizzBuzz.java
public class FizzBuzz{
	public static void main(String[] args){
		for(int n = 1; n<=100; n++){
			if(n%15==0){System.out.println("FIZZBUZZ (" + n +")");}
			else if(n%3==0){System.out.println("FIZZ (" + n +")");}
			else if(n%5==0){System.out.println("BUZZ (" + n +")");}
			else{System.out.println(n);}
		}
	}
}

File: FizzBuzz.rb
#!/usr/bin/env ruby

class FizzBuzz
	n = 1
	while n <= 100 do
		if n%15==0
			puts "FIZZBUZZ (#{n})"
		elsif n%3==0
			puts "FIZZ (#{n})"
		elsif n%5==0
			puts "BUZZ (#{n})"
		else puts n
		end
		n+=1
	end
end